Sources/XCUITestHelpers/XCUIElement+Visibility.swift

```diff
@@ -0,0 +1,10 @@
+import XCTest
+
+public extension XCUIElement {
+
+    func isFullyVisibleOnScreen(in app: XCUIApplication = XCUIApplication()) -> Bool {
+        guard exists && frame.isEmpty == false && isHittable else { return false }
```

Comment:
I like the move from `!frame.isEmpty` to `frame.isEmpty == false`, much better to read.

Reply:
Thanks. I once heard that prefixing with `!` risks going unnoticed when a reader skims the code, which I found plausible. This syntax is more verbose but harder to miss. 😄